symbolic-examples/symbolic-examples.cabal

```diff
@@ -127,6 +134,10 @@ library
 
 benchmark bench-compiler-performance
     import:           options-exe
+    if arch(wasm32)
+      buildable: False
```

Comment:
executables mess with compilation to wasm

Reply:
maybe this can be specified in `common options-exe` then?